build/NuSpecs/Microsoft.WindowsAppSDK.Bootstrap.CS.targets

```diff
@@ -1,7 +1,23 @@
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003"> 
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <Target Name="GenerateBootstrapCS" 
-            BeforeTargets="BeforeCompile">
+    <PropertyGroup Condition="'$(WindowsAppSDKBootstrapAutoInitialize)'==''">
+        <WindowsAppSDKBootstrapAutoInitialize>true</WindowsAppSDKBootstrapAutoInitialize>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(WindowsAppSDKBootstrapAutoInitialize)'=='true'">
+        <DefineConstants
+        <DefineConstants Condition="'$(WindowsAppSDKBootstrapAutoInitializeOptions_Default)'=='true'">MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_DEFAULT</DefineConstants>
+        <DefineConstants Condition="'$(WindowsAppSDKBootstrapAutoInitializeOptions_None)'=='true'">MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_DEFAULT</DefineConstants>
```

Comment:
None also maps to Default?  typo?

Reply:
Yep copy/paste error. See other comment. Fixed. Thanks.